package.json

```
@@ -92,13 +92,14 @@
     "build-js": "npx webpack",
     "build-js:watch": "npx webpack --mode=development --watch",
     "build-demo-module": "npx esbuild src/BookReaderTemplate/BookReaderTemplate.js --outfile=BookReaderDemo/bookreader-template-bundle.js --bundle",
+    "build-demo-watch": "npx esbuild src/BookReaderTemplate/BookReaderTemplate.js --outfile=BookReaderDemo/bookreader-template-bundle.js --bundle --watch",
     "build-css": "npx sass --no-source-map ./src/css/BookReader.scss ./BookReader/BookReader.css",
     "build-css:watch": "npx sass --watch --no-source-map ./src/css/BookReader.scss ./BookReader/BookReader.css",
     "lint": "npx eslint src/js tests",
     "lint:fix": "npx eslint --fix src/js tests",
     "serve": "npx http-server . --port 8000",
     "serve-live": "npx live-server . --port 8000 --watch=index.html,BookReader,BookReaderDemo",
-    "serve-dev": "npx concurrently --kill-others npm:serve-live npm:build-*:watch",
+    "serve-dev": "npx concurrently --kill-others npm:build-demo-watch npm:serve-live npm:build-*:watch",
```

Then it'll be cause by the `*:watch` :)

```suggestion
    "serve-dev": "npx concurrently --kill-others npm:serve-live npm:build-*:watch",
```

---

ha! good catch. I saw that first and thought it would solve all of my problems but unfortunately it can't find "import lit-element" if it watches after the server starts